src/wp-includes/widgets.php

```diff
@@ -1042,19 +1042,29 @@ function wp_get_sidebars_widgets( $deprecated = true ) {
  * @access private
  *
  * @global array $_wp_sidebars_widgets
- * @param array $sidebars_widgets Sidebar widgets and their settings.
+ * @global array $sidebars_widgets
+ *
+ * @param array   $new_sidebars_widgets            Sidebar widgets and their settings.
+ * @param boolean $refresh_global_sidebars_widgets Optional. Whether to update $sidebars_widgets
+ *                                                 global. Default true.
  */
-function wp_set_sidebars_widgets( $sidebars_widgets ) {
-	global $_wp_sidebars_widgets;
+function wp_set_sidebars_widgets( $new_sidebars_widgets, $refresh_global_sidebars_widgets = true ) {
```

Comment:
Even though the second parameter is not required, if someone wishes to override `$refresh_global_sidebars_widgets` to `false`, they would need to verify the version is > 5.8.1. I think addressing this in 5.9 and allowing the simplified > 5.9 check (or allowing a plugin to raise the minimum version of WP required to 5.9) would be preferred.

Also, there should be a `@since` annotation in the doc block explaining what changed and **when.

Reply:
> Also, there should be a @since annotation in the doc block explaining what changed and **when.

I just added one! 

I'll resolve this conversation since choosing the version number is now discussed in the the "main" thread.